mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/json/IndexableMessage.java

```diff
@@ -98,6 +101,17 @@ private static ZonedDateTime getSanitizedInternalDate(MailboxMessage message, Zo
             zoneId);
     }
 
+    private void generateText() {
+        this.text = Stream.of(from.serialize(),
```

Comment:
I would have put this code in getText() to ensure we never forget to build the field